INSTALL.md

```diff
@@ -27,43 +27,62 @@ Use `make dependencies` to map the native libraries to your system and fetch the
 
 To compile OpenRA, run `make all` from the command line.
 
-Run with either `launch-game.sh` or `mono --debug OpenRA.Game.exe`.
+Run with either `launch-game.sh` or `mono --debug OpenRA.Game.exe`. It is also possible to specify the mod you wish to run from the command line, e.g. with `./launch-game.sh Game.Mod=ts` if you wish to try the experimental Tiberian Sun mod. 
```

Comment:
Can we drop the OpenRA.Game.exe option here? There is no reason for normal players to not be using the launcher script.

Reply:
Sure, only left it in there because it was there already.

Reply:
Should I also remove it for macOS while I'm at it?

Reply:
Yeah, sure